robottelo/ui/base.py

```diff
@@ -686,6 +686,11 @@ def select(self, target, list_value, wait_for_ajax=True, timeout=30,
         if self.element_type(target) == 'select':
             if isinstance(target, tuple):
                 element = self.wait_until_element(target)
+                if element is None:
```

Comment:
Just do what @abalakh suggested and try to avoid such situations in future (like we have in title `StaleElementException`, but nothing about it in the content)